app/assets/javascripts/pages.home.js

```diff
@@ -39,6 +40,8 @@ app.controller('HomePageController', ['$scope','$http', '$uibModal', function($s
       }
     }
 
+    $scope.getHosts(1);
```

Comment:
in the top of the html, the function `init` is called. make sure they don't conflict or do a double call